tfs/src/main/java/hudson/plugins/tfs/rm/ReleaseManagementCI.java

```diff
@@ -163,6 +233,9 @@ void CreateRelease(
             JSONObject object = new JSONObject(response);
             listener.getLogger().printf("Release Name: %s%n", object.getString("name"));
             listener.getLogger().printf("Release id: %s%n", object.getString("id"));
+            String logLink = this.collectionUrl + this.projectName
+                    + "/_release?releaseId=" + object.getString("id") + "&_a=release-logs";
```

Comment:
Please do not construct the url by hand. In the release object under _link property you will notice "self" and 'web" keys. Use the url under the "web" key.